src/woo_search/search_index/api/serializers/publications.py

```diff
@@ -106,16 +111,24 @@ def validate(self, attrs: DocumentIndexType):
 
 class PublicationSerializer(serializers.Serializer):
     uuid = serializers.CharField()
-    publisher = PublisherSerializer(
+    publisher = NestedPublisherSerializer(
         help_text=_("The organisation which publishes the publication.")
     )
-    informatie_categorieen = InformatieCategorieSerializer(
+    informatie_categorieen = NestedInformationCategorySerializer(
         help_text=_(
             "The information categories clarify the kind of information present in the publication."
         ),
         required=True,
         many=True,
     )
+    onderwerpen = NestedTopicSerializer(
+        help_text=_(
+            "Topics capture socially relevant information that spans multiple publications. "
+            "They can remain relevant for tens of years and exceed the life span of a single publication."
+        ),
+        required=True,
```

Comment:
this also technically makes it a breaking change requiring a major version bump of the API. I don't see why this field must be required tbh, we can get by perfectly fine with it being empty/absent.